integrationtests/Paket.IntegrationTests/App.config

```diff
@@ -9,15 +9,10 @@
     <assemblyIdentity name="FSharp.Core" publicKeyToken="b03f5f7f11d50a3a" culture="neutral" />
     <bindingRedirect oldVersion="0.0.0.0-65535.65535.65535.65535" newVersion="5.0.0.0" />
   </dependentAssembly>
-  <dependentAssembly>
```

Comment:
some of these app.configs got changed a lot after adding the new nuget.packaging package....